adafruit_fingerprint.py

```diff
@@ -25,7 +25,12 @@
   https://github.com/adafruit/circuitpython/releases
 """
 
-from micropython import const
+try:
```

Comment:
I'm curious why this was added.

Reply:
I'm running these on Windows PC so needed a way to get around the micropython requirements. Any suggestion on how to get around it?

Reply:
I'm not sure I understand - I thought the micropython modue was satisfied but the Blinka interface -- at least, it is on a Raspberry Pi. 
Does the example work an a Windows system?

Reply:
Okay, I looked into this further, the issue was that my venv was missing Blinka. I thought setup had installed but apparently not. I removed the pseudo code. The examples work on both Linux (Ubuntu) and Windows 10.